dexmaker/src/main/java/com/android/dx/stock/ProxyBuilder.java

```diff
@@ -41,7 +41,6 @@
 import java.util.HashSet;
```

Comment:
I think you still have to change the implementation of Dexmaker#generateFileName to also consider the interfaces

Reply:
I'm not sure why we should do that to solve #124. Could you please explain the reason?

Reply:
Changed generateFileName() to consider interfaces